src/sentry/tasks/reports.py

```diff
@@ -554,21 +554,21 @@ def prepare_organization_report(timestamp, duration, organization_id, dry_run=Fa
 
 def fetch_personal_statistics(start__stop, organization, user):
     start, stop = start__stop
-    resolved_issue_ids = Activity.objects.filter(
-        project__organization_id=organization.id,
-        user_id=user.id,
-        type__in=(Activity.SET_RESOLVED, Activity.SET_RESOLVED_IN_RELEASE, ),
-        datetime__gte=start,
-        datetime__lt=stop,
-        group__status=GroupStatus.RESOLVED,  # only count if the issue is still resolved
-    ).distinct().values_list(
```

Comment:
Weird, I thought values_list gave you a regular array, didn't realize it was a ValuesListQuerySet